lib/ansible/module_utils/urls.py

```diff
@@ -606,11 +606,11 @@ def get_ca_certs(self):
                     full_path = os.path.join(path, f)
                     if os.path.isfile(full_path) and os.path.splitext(f)[1] in ('.crt','.pem'):
                         try:
-                            cert_file = open(full_path, 'r')
+                            cert_file = open(full_path, 'br')
```

Comment:
It seems that standard style in Ansible is "rb" rather than "br" (primary mode first, modifiers afterwards) but it's just style, not a blocker.